### PR TITLE
Molecule Test Fabric( storageclass, namespaces, service accounts )

### PR DESCRIPTION
**Pull request by suvajit-sarkar**

### CODE REVIEW

File: platforms/hyperledger-fabric/configuration/molecule/default/converge.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
+---
+- name: Converge
+  hosts: localhost
+  connection: local
+  vars:
+    privilege_escalate: true            #escalate root priviledge for default test
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/molecule/default/converge.yml:6]: this has to be false

File: platforms/hyperledger-fabric/configuration/molecule/default/converge.yml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        release_dir: "{{ playbook_dir }}/../../../releases/dev"
+        sc_name: awsordererstorageclass
+        result:
+          resources: ""
+        item:
+          cloud_provider: aws
+        org:
+          name: test
+          cloud_provider: aws
+          k8s:
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/molecule/default/converge.yml:25]: common vars across all tasks should be in the main vars, no need to redefine

File: platforms/hyperledger-fabric/configuration/molecule/default/converge.yml [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        state: present
+        src: "{{ playbook_dir }}/../../../releases/dev/orderer/awsordererstorageclass.yaml"
+    - name: Deploy the storageclass on cluster
+      k8s:
+        state: present
+        src: "{{ playbook_dir }}/../../../releases/dev/peer/awspeerstorageclass.yaml"
+    - name: Deploy the storageclass on cluster
+      k8s:
+        state: present
+        src: "{{ playbook_dir }}/../../../releases/dev/orderer/azureordererstorageclass.yaml"
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/molecule/default/converge.yml:105]: Does the azure and aws storageclass actually get created?

File: platforms/hyperledger-fabric/configuration/molecule/default/verify.yml [hunk 4]
```diff
@@ -0,0 +1,195 @@
+---
+    - name: Verify
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/molecule/default/verify.yml:2]: too many tabs at the start

File: platforms/hyperledger-fabric/configuration/roles/create/anchorpeer/tasks/main.yaml [hunk 5]
```diff
@@ -10,4 +10,4 @@
   loop: "{{ participants }}"
   loop_control:
     loop_var: participant
-
+
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/roles/create/anchorpeer/tasks/main.yaml:13]: remove spaces.

File: platforms/hyperledger-fabric/configuration/molecule/default/converge.yml (hunk 1 above)
Comment (suvajit-sarkar) [reply platforms/hyperledger-fabric/configuration/molecule/default/converge.yml:6]: ok

**Pull request merged**